src/js/media-preview-thumbnail.js

```diff
@@ -82,7 +76,6 @@ class MediaPreviewThumbnail extends window.HTMLElement {
   attributeChangedCallback(attrName, oldValue, newValue) {
     if (
       [
-        Attributes.TIME,
```

Comment:
removed `Attributes.TIME` because it didn't do anything

Reply:
Currently, it isn't possible to force the preview thumbnail to show a particular thumbnail. Maybe `Attributes.TIME` was supposed to do that? The comment at the top of the file seems to imply that.
I think such a feature would be useful, but this isn't a blocker or req for v1.

Could you update the comment at the top of the file (or just delete it)